sql/hive/src/main/scala/org/apache/spark/sql/hive/client/ClientWrapper.scala

```diff
@@ -252,7 +252,7 @@ private[hive] class ClientWrapper(
     false
   }
 
-  def client: Hive = {
+  override def client: Hive = {
```

Comment:
i just change `override def client: Hive ` to `def client: Hive`, and run "sbin/start-thriftserver.sh" successfully. But run the app failed. My code of app is :
```
import java.sql.DriverManager

object JDBCExperiments {
  def main(args: Array[String]) {
    val url = "jdbc:hive2://localhost:10000/default"
    val username = "lian"
    val password = ""

    try {
      Class.forName("org.apache.hive.jdbc.HiveDriver")
      val connection = DriverManager.getConnection(url, username, password)
      val metadata = connection.getMetaData
      val tables = metadata.getTables(null, null, null, null)
      while (tables.next()) {
        val fields = Array.tabulate(5) { i =>
          tables.getString(i + 1)
        }
        println(fields.mkString(", "))
      }

      val columns = metadata.getColumns(null, null, null, null)
      while (columns.next()) {
        println((columns.getString(3), columns.getString(4), columns.getString(6)))
      }
    }
  }
}
```
The error info is :
```
Caused by: java.sql.SQLException: Failed to start database 'metastore_db' with class loader sun.misc.Launcher$AppClassLoader@799dbc3b, see the next exception for details.
        at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
        at org.apache.derby.impl.jdbc.SQLExceptionFactory40.wrapArgsForTransportAcrossDRDA(Unknown Source)
        ... 114 more
Caused by: java.sql.SQLException: Another instance of Derby may have already booted the database /home/hll/code/datasight-spark/metastore_db.
        at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
        at org.apache.derby.impl.jdbc.SQLExceptionFactory40.wrapArgsForTransportAcrossDRDA(Unknown Source)
        at org.apache.derby.impl.jdbc.SQLExceptionFactory40.getSQLException(Unknown Source)
        at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
        ... 111 more
Caused by: ERROR XSDB6: Another instance of Derby may have already booted the database /home/hll/code/datasight-spark/metastore_db.
```